pkg/schedule/windowed_schedule_test.go

```diff
@@ -14,11 +14,55 @@ import (
 func TestWindowedSchedule(t *testing.T) {
 	log.SetLevel(log.DebugLevel)
 	Convey("Windowed Schedule", t, func() {
+		Convey("nominal window without misses", func() {
+			startWait := time.Millisecond * 50
+			windowSize := time.Millisecond * 200
+			interval := time.Millisecond * 10
+
+			start := time.Now().Add(startWait)
+			stop := time.Now().Add(startWait + windowSize)
+			w := NewWindowedSchedule(
+				interval,
+				&start,
+				&stop,
+			)
+
+			err := w.Validate()
+			So(err, ShouldBeNil)
+
+			var r []Response
+			last := *new(time.Time)
+
+			state := Active
+			before := time.Now()
+			for state == Active {
+				r1 := w.Wait(last)
+				state = r1.State()
+				last = time.Now()
+				r = append(r, r1)
+			}
+			// we should have either 21 or 23 minus 0 missed
```

Comment:
Yes, I see. I changed the comment through all tests to be more appropriate. Now for each of tests cases, window-width is 200ms, so for interval 10ms it is expected to get in a range 19 - 22 responses (or 15-18 for tests when there are some missed responses). 
@jcooklin, could you run those tests on your system to confirm that?